docs/src/modules/components/DiamondSponsors.js

```diff
@@ -95,14 +95,14 @@ export default function DiamondSponsors() {
           data-ga-event-category="sponsor"
           data-ga-event-action="drawer"
           data-ga-event-label="doit"
-          href="https://www.doit-intl.com/flexsave/?utm_source=materialui&utm_medium=referral"
+          href="https://www.doit.com/flexsave/?utm_source=materialui&utm_medium=referral"
           rel="noopener noreferrer sponsored"
           target="_blank"
         >
           <img
             height="26"
-            src="/static/sponsors/doit.svg"
-            alt="doit-intl"
+            src={`/static/sponsors/doit-${theme.palette.mode}.svg`}
```

Comment:
Shouldn't it be the other way around? It currently shows the `doit-light` logo (with white letters) on a white background (see https://deploy-preview-35030--material-ui.netlify.app/material-ui/getting-started/overview/)

Reply:
That's correct, thanks for the spot.
Fixed.